vue/wolox-books/src/views/Register.vue

```diff
@@ -72,6 +83,9 @@ export default {
           }
         }
       }
+    },
+    goLogin () {
+      this.$router.push('/login')
```

Comment:
Maybe you'd like to save the token here

Reply:
Ok, but in this card does not ask save token,  I think is on coming cards